seatunnel-connectors/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/flink/sink/DruidSink.java

```diff
@@ -45,14 +47,16 @@
     private String timestampFormat;
     private String timestampMissingValue;
 
+    @Nullable
     @Override
-    public DataSink<Row> outputBatch(FlinkEnvironment env, DataSet<Row> dataSet) {
-        DataSink<Row> dataSink = dataSet.output(new DruidOutputFormat(coordinatorURL, datasource, timestampColumn, timestampFormat, timestampMissingValue));
+    public DataStreamSink<Row> outputStream(FlinkEnvironment env, DataStream<Row> dataStream) {
+        DataStreamSink<Row> rowDataStreamSink = dataStream.addSink(new DruidSinkFunction<>(
+                new DruidOutputFormat<>(coordinatorURL, datasource, timestampColumn, timestampFormat, timestampMissingValue)));
         if (config.hasPath(PARALLELISM)) {
             int parallelism = config.getInt(PARALLELISM);
-            return dataSink.setParallelism(parallelism);
+            rowDataStreamSink.setParallelism(parallelism);
         }
-        return dataSink;
+        return null;
```

Comment:
May I ask why we don't return the sink instance here

Reply:
> 

You are right.

Reply:
I can change this function to void and then submit this pr again.

Reply:
I'm done. @leo65535 @asdf2014

Reply:
@xsaffable Thanks for your contribution, let's wait the CI pass. Sorry, I don't have permission to help you to start the pending workflows. Maybe @leo65535 can help you out I think.

Reply:
Thanks for your review @asdf2014

Reply:
@leo65535 You are welcome

Reply:
Thank you @leo65535 @asdf2014 . But I see it seems to time out, can you help me